.gitignore

```diff
@@ -12,7 +12,16 @@ debian/kodi-pvr-iptvsimple.substvars
 debian/kodi-pvr-iptvsimple/
 debian/tmp/
 obj-x86_64-linux-gnu/
+CMakeCache.txt
```

Comment:
The idea is that you build "out of tree" in a separate `build` directory, then you only need to ignore that directory. Can you do that?